### PR TITLE
Simplify and fix countdown display skipping seconds

### LINKED ISSUE #90: Timer skips seconds
**Describe the bug**
The timer skips seconds from time to time <sup>(pun intended)</sup>. For example, it may go from 53 to 51 without seeing 52.

**To Reproduce**
1. Set timer to 1 minute
2. Watch it count down and observe it skipping numbers

**Expected behavior**
It shouldn't skip numbers.

**Desktop (please complete the following information):**
 - OS version: macOS 10.15.5
 - Version: 1.5.4

### PR DESCRIPTION
**Pull request by robinstewart**
I was able to fix #90 and also simplify the tick timer code by using round() instead of floor(). Since the UI only allows timers to be set to an integer number of seconds, the exact "seconds remaining" is always near an integer boundary. We can simply use the `round()` function to make sure the nearest integer is displayed, regardless of whether the timer ends up firing slightly before or slightly after.

Here is a typical run when I uncomment the print statement:

    9.992520928382874
    8.992309927940369
    7.992242932319641
    6.992233991622925
    5.991078019142151
    4.992653012275696
    3.9753549098968506
    2.979225993156433
    1.9741169214248657
    0.9946630001068115
    -0.010595083236694336

Fixes #90
Closes #93
Closes #92
Closes #91

### CODE REVIEW

**Comment by luckman212** [comment]
@robinstewart I like your `round()` solution. Very nice! Tested & works for me. I even tested it with tolerances up to `0.25` with no missed ticks. But, I don't think those extra milliseconds of CPU time saved are worth it — `0.03` is good 👍

File: Timer/MVClockView.swift [hunk 1]
```diff
@@ -428,26 +432,16 @@ class MVClockView: NSControl {
...
-    currentTimeTimer?.tolerance = 0.03
-  }
-
   @objc func tick() {
     guard let timerTime = self.timerTime  else { return }
 
-    self.seconds = fmax(0, floor(CGFloat(timerTime.timeIntervalSinceNow)))
-    if self.seconds <= 0 {
+    let secondsRemaining = CGFloat(timerTime.timeIntervalSinceNow)
+    //print(secondsRemaining)
```
Comment (karbassi) [inline Timer/MVClockView.swift]: Can you remove this?

**Comment by karbassi** [comment]
@robinstewart looks good to me as well.

Merging

Fixes #90 
Closes #93 
Closes #92 
Closes #91

**Comment by robinstewart** [comment]
Thanks!

**Pull request merged**